core/src/main/resources/reference.conf

```diff
@@ -57,6 +57,15 @@ data-source {
   }
 }
 
+slick {
+  # batchSize is used for writes, to group inserts/updates
+  # this must be explicitly utilized via custom business logic
+  batchSize = 2000
+  # fetchSize is used during Slick streaming to set the size of pages
+  # this must be explicitly set via withStatementParameters
+  fetchSize = 5000
+}
+
```

Comment:
I'll need to add `fetchSize` (and associated comments) to firecloud-develop as well. At least for this PR they're just here in reference.conf